.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+
+language: ruby
+rvm:
+  - 2.3.3
+
+services:
+  - docker
+
+before_install:
+  - cd angular-rails-app && docker build -t rails-app .
+  - docker ps
+
```

Comment:
Default install runs bundler locally. We should override it to ensure bundle only runs in the docker container.